main.go

```diff
@@ -281,6 +265,23 @@ func validateParams() {
 	}
 }
 
+func handler(c *controller, ns *v1.Namespace) error {
+	log.Print("Refreshing credentials...")
+	secrets := c.generateSecrets()
+	for _, secret := range secrets {
+		if *argSkipKubeSystem && ns.GetName() == "kube-system" {
+			continue
+		}
+
+		if err := c.processNamespace(ns, secret); err != nil {
+			return err
```

Comment:
This is where I think a `continue` could be appropriate, as mentioned  [above](https://github.com/upmc-enterprises/registry-creds/pull/40/files?diff=split#r104818230), though I don't think continuing the loop will result in a different outcome for a different provider.